common-javassist/src/main/java/io/servicecomb/common/javassist/JavassistUtils.java

```diff
@@ -164,6 +178,37 @@ private static void addEnumValuesMethod(CtClass ctClass, List<String> values) th
         }
     }
 
+    public static String capitalize(String name) {
+        if (name == null || name.length() == 0) {
+            return name;
+        }
+        return name.substring(0, 1).toUpperCase(ENGLISH) + name.substring(1);
```

Comment:
copy from java.beans.NameGenerator.capitalize(String)
NameGenerator is package protected, can not be called.
if name.length == 1 will not throw exception